src/components/common/FileUploadPreview.vue

```diff
@@ -88,25 +57,47 @@ export default class FileUploadPreview extends Mixins(DocumentMixin) {
 
   /** Programmatically opens the file selection dialog. Can be called externally. */
   public clickFileInput (): void {
+    // first clear any existing file
+    this.fileUpload = null
```

Comment:
This is needed or this component retains the previously uploaded file and would do nothing if we tried to upload this file again ... we could not identify consecutive duplicates.